avroconsumer/src/docker/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
```

Comment:
In your case it is enough to have only one docker-compose.yml file. Move docker directory to the project root and delete the rest docker-compose.yml files

Reply:
Done